__tests__/app.test.js

```diff
@@ -127,3 +127,22 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status: 200, should responds with array of comments for a given article", () => {
+    return request(app)
+      .get(`/api/articles/1/comments`)
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
I'd add a check for the length here to prevent any false positives if the array was empty!